### ISSUE TITLE
Empty element in OutlinePane via XML causes error

### ISSUE THREAD

**Issue by stefvanschie** [stefvanschie/IF#88]
When adding an empty element to an outline pane in an XML file, this causes an exception to occur when this XML file is being loaded. This is because gui items don't accept air items, since they can't store persistent data.

A possible workaround would be to not set the persistent data for air items. This is unnecessary to begin with, since air items cannot be moved and therefore do not need to be tracked.

Example code that causes such an error:
```xml
<gui title="test" rows="1" type="chest">
  <outlinepane x="0" y= "0" length="1" height="1">
    <empty />
  </outlinepane>
</gui>
```

Stacktrace: https://pastie.io/qgvtbz.txt

**Comment by stefvanschie** [comment]
This will be fixed in 0.9.0.

**Comment by stefvanschie** [comment]
This issue was reintroduced in version 0.10.5, after `InventoryComponent#setItem` started copying items instead of using the items directly. This is to ensure that two identical `ItemStack` instances with different actions don't end up having the same action. However, copying a `GuiItem` fails for air items, meaning an XML file with the empty tag will now fail to open. `GuiItem#copy` should accept air items.

**Comment by stefvanschie** [comment]
This will be fixed in version 0.10.16.
